packages/lit-html/src/lib/lit-html.ts

```diff
@@ -563,9 +565,8 @@ export class NodePart {
 
   private __commitText(value: unknown): void {
     const node = this.__startNode.nextSibling;
-    // If `value` isn't already a string, we explicitly convert it here in case
-    // it can't be implicitly converted - i.e. it's a symbol.
-    value = value == null ? '' : value;
+    // If `value` isn't already a string, we explicitly convert it here
+    value = value ??= '';
```

Comment:
This should just be 

```suggestion
    value ??= '';
````